clara/py_parser.py

```diff
@@ -85,6 +85,9 @@ def visit_Num(self, node):
     def visit_Str(self, node):
         return Const('"{}"'.format(node.s), line=node.lineno)
 
+    def visit_NameConstant(self, node):
+        return Const('"{}"'.format(node.s), line=node.lineno)
```

Comment:
Won't this create a string constant?

Reply:
Let me check later. Maybe it should return a boolean value instead of string. 🤔

Reply:
I think that `NameConstant` can be: `"True"`, `"False"` or `"None"`. So, I think that `node.s` should be passed to `Const` as a first argument instead of `'"{}"'.format(node.s)`. However, I would add the very minimal tests to check this.